### ISSUE TITLE
Single underscore not detected as unused in import statement

### ISSUE THREAD

**Issue by danielrentz** [microsoft/TypeScript#45967]
# Bug Report

### 🔎 Search Terms

unused underscore

### 🕗 Version & Regression Information

- This is the behavior in every version I tried.

### ⏯ Playground Link

https://www.typescriptlang.org/play?noUnusedParameters=true#code/FASwtgDg9gTgLgAgPoIGYymBAiArgOwBMBTGAZwGNZjsBuUSWRAEjQy2wCsBHXUgTzrAgA

### 💻 Code

```ts
import _ from "underscore";
import $ from "jquery";
```

### 🙁 Actual behavior

The JQuery import is marked as unused, but the Underscore import is not.

### 🙂 Expected behavior

A single `_` is not treated as an identifier with leading "unused marker", but as regular identifier.

**Comment by MartinJohns** [comment]
> 🙂 Expected behavior  
> A single `_` is not treated as an identifier with leading "unused marker", but as regular identifier.

But a single `_` is a **very** common identifier for unused parameters.

**Comment by danielrentz** [comment]
Ok... I would be fine with restricting my request to `import` statements. There, a `_` is a very common identifier for Underscore/Lodash ;)

**Comment by FilipeBeck** [comment]
It sounds like a bug. There is no reason to use `_` as an identifier to avoid errors with _unused vars_. If you just want the side effects, can use `import 'module'`.

**Comment by fabiospampinato** [comment]
I'd love to see this fixed, I must have dozens of no longer needed "_ from 'lodash'" imports that TS seemingly can't see.

**Comment by ethanresnick** [comment]
By the way, this applies even if the binding starts with an underscore, but isn't literally the underscore character. E.g., I had imports of a function whose name started with `__x`, and those weren't flagged as unused.

I would definitely not want TS to change its interpretation of the leading underscore in non-import contexts. E.g., when defining a callback or destructuring an array, it's common to want to skip one of arguments/elements, and using a binding with a leading underscore is great for that. 

But, in the case of an _import_, there's no practical use case I can think of for wanting to suppress an "unused identifier" warning. So, this behavior feels like a bug — the sort of thing that arises inadvertently from global application of the "leading underscore = suppress unused binding error" — rather than behavior anyone would actually want.
